src/components/uielements/ContentImage.astro

```diff
@@ -0,0 +1,28 @@
+---
+import type { ImageMetadata } from 'astro';
+import { Image } from 'astro:assets';
+
+export interface Props {
+    src: string,
+    imageFolder?: string,
+    alt?:string,
+    loading?: "lazy" | "eager" | null | undefined,
+    class?: string,
+    width?: number,
+    height?: number,
+}
+
+const { 
+    class: className, 
+    src: srcFile,
+    imageFolder = '/src/assets/images/*.{jpeg,jpg,png,gif}',
+    alt: altText = '',
+    width,
+    height,
+    loading = 'lazy',
+} = Astro.props;
+
+const images = import.meta.glob<{ default: ImageMetadata }>('/src/assets/images/*.{jpeg,jpg,png,gif}');
```

Comment:
Kunt gij straks anders eens proberen voor de subfolders te laten werken? ik denk dat het dan 
const images = import.meta.glob<{ default: ImageMetadata }>('/src/assets/images/**/*.{jpeg,jpg,png,gif}'); moet zijn, voor nu is em goed